app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -91,12 +91,4 @@ class MainFragment : Fragment() {
             }
         }
     }
-
-    fun getClickedStock(view: View) {
-        Log.d("[SEARCH BAR]", "Clicked a stock.")
-    }
-
```

Comment:
This small function was unused by anything else so I removed it. https://www.oreilly.com/library/view/becoming-a-better/9781491905562/ch04.html